common/MultiTarget/TargetFrameworks.props

```diff
@@ -0,0 +1,22 @@
+<Project>
```

Comment:
Was this not moved from somewhere?

Reply:
This file was one directory up, in the `./common` folder. It pertains to targeting the different platforms, so it was moved here.

Reply:
@Arlodotexe right, so shouldn't we see it as a move and not a new file? I don't know if I saw the delete in the change list?

Reply:
Interesting, I'll take a look